CMakeLists.txt

```diff
@@ -12,7 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.25 FATAL_ERROR)
+if (WIN32)
+  # Use a newer version of CMake on Windows, so we are NEW for
+  # https://cmake.org/cmake/help/latest/policy/CMP0149.html
+  cmake_minimum_required(VERSION 3.27 FATAL_ERROR)
+else()
+  cmake_minimum_required(VERSION 3.25 FATAL_ERROR)
```

Comment:
If we need a newer version on Windows, why not require a newer version on Linux/Mac, too?

Reply:
It might break platforms that have 3.25 but don't have 3.27. I know it is a narrow range, but I don't want to need to spend time chasing it down. We can always remove the conditional later.